src/main/java/games/strategy/engine/data/properties/ComboProperty.java

```diff
@@ -20,36 +20,20 @@
   private T value;
 
   /**
-   * @param name
-   *        name of the property.
-   * @param defaultValue
-   *        default string value
-   * @param possibleValues
-   *        collection of values
+   * @param name name of the property.
+   * @param defaultValue default string value
+   * @param possibleValues collection of values
    */
-  public ComboProperty(final String name, final String description, final T defaultValue,
+  public ComboProperty(final String name,
+      final String description,
+      final T defaultValue,
       final Collection<T> possibleValues) {
-    this(name, description, defaultValue, possibleValues, false);
-  }
-
-  @SuppressWarnings("unchecked")
-  private ComboProperty(final String name, final String description, final T defaultValue,
-      final Collection<T> possibleValues, final boolean allowNone) {
     super(name, description);
-    if (!allowNone && !possibleValues.contains(defaultValue) && defaultValue == null) {
+    this.possibleValues = new ArrayList<>(possibleValues);
+    if (defaultValue == null && !possibleValues.contains(defaultValue)) {
```

Comment:
I'm not quite sure what this existing code was checking.  Should it  be `defaultValue != null`?  Otherwise, it's looking for `null` in `possibleValues`, which doesn't seem right.

Reply:
I believe this is just fine:
If the default value is null, and null is not a possible value, and exception is being thrown 🤷‍♂️

Reply:
The corollary is then if `defaultValue` is not `null`, it doesn't need to be present in `possibleValues`?  The exception message seems to reveal the original intent:

```
possible values does not contain default
```

Just seems backwards in the early AM here. :smile:

Reply:
Makes sense, however I'm not going to dig into any details for this PR ^^
We can check if null is every being passed to this method, but in a follow-up PR ^^